core/src/main/scala-2.13+/cats/data/NonEmptyLazyList.scala

```diff
@@ -213,7 +213,7 @@ class NonEmptyLazyListOps[A](private val value: NonEmptyLazyList[A])
    * Tests if some element is contained in this NonEmptyLazyList
    */
   final def contains(a: A)(implicit A: Eq[A]): Boolean =
-    toLazyList.contains(a)
+    toLazyList.exists(A.eqv(_, a))
```

Comment:
this looks like a straight bug. The contains method was ignoring the A. This is technically changing behavior, but I would argue it is a bug fix.